core/citrus-spring/src/main/java/com/consol/citrus/config/CitrusNamespaceParserRegistry.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2006-2010 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.consol.citrus.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.consol.citrus.config.xml.*;
+import com.consol.citrus.spi.ResourcePathTypeResolver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.xml.BeanDefinitionParser;
+
+/**
+ * Registers bean definition parser for beans in test case, handled by {@link com.consol.citrus.config.handler.CitrusTestcaseNamespaceHandler}
+ *
+ * @author Christoph Deppisch
+ * @since 2007
+ */
+public final class CitrusNamespaceParserRegistry {
```

Comment:
GitHub does not seem to notice that this has been renamed from `core/citrus-spring/src/main/java/com/consol/citrus/config/TestActionRegistry.java`. have there been any changes or is it just the renaming?